internal/zinc-classfile/src/main/scala/sbt/internal/inc/classfile/Analyze.scala

```diff
@@ -45,16 +42,16 @@ private[sbt] object Analyze {
     for (
       newClass <- newClasses;
       classFile = Parser(newClass);
-      sourceFile <- classFile.sourceFile orElse guessSourceName(newClass.getName);
+      _ <- classFile.sourceFile orElse guessSourceName(newClass.getName);
       source <- guessSourcePath(sourceMap, classFile, log);
       binaryClassName = classFile.className;
       loadedClass <- load(binaryClassName, Some("Error reading API from class file"))
     ) {
       binaryClassNameToLoadedClass.update(binaryClassName, loadedClass)
-      val srcClassName = binaryToSourceName(loadedClass)
+      val maybeSrcClassName = binaryToSourceName(loadedClass)
```

Comment:
No problem, I did this because few lines below we are reusing name `srcClassName` inside `match` -> I renamed the value inside the match to avoid confustion